Directory.Packages.props

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
     <!-- Version of the Microsoft.Identity.Client.NativeInterop package. -->
-    <MSALRuntimeNativeInteropVersion>0.13.14</MSALRuntimeNativeInteropVersion>
+    <MSALRuntimeNativeInteropVersion>0.14.1</MSALRuntimeNativeInteropVersion>
```

Comment:
@ashok672 - is this ok?

CC @gladjohn

Reply:
also does anyone what repo Microsoft.Identity.Client.NativeInterop comes from?

Reply:
It's a non-public repo. What changes would we have to make there?

Reply:
according to this https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4495 it has DisableOptimizations set in their DebuggableAttribute.

Reply:
I would let @ashok672 confirm this

Reply:
@ashok672 bump